### PR TITLE
Add 2023 Q2 Hyperledger Cacti Project Report

### PR DESCRIPTION
**Pull request by takeutak**

### CODE REVIEW

File: project-reports/2023/2023-Q2-Hyperledger-Cacti.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+# 2023 Q2 Hyperledger Cacti
+
+## Project Health
+
+- The first code merge of Cactus and Weaver to "Cacti" repository was completed and we released Cacti v1.2.0 in the end of March.
+- The maintainers are also discussing the [new architecture of Cacti](https://github.com/hyperledger/cacti/blob/main/ROADMAP.md#cacti-v2) toward Cacti-V2 in order to combine the strengths of the original Cactus and Weaver systems to provide a more flexible interoperablity tool with several feature selection options given to network administrators.  The Cacti v2-alpha, v2-beta and v2.0 release will be redesigned according to the new architecture.
+
+# Questions/Issues for the TSC
```
Comment (tkuhrt) [inline project-reports/2023/2023-Q2-Hyperledger-Cacti.md]:
```suggestion
# Questions/Issues for the TOC
```

File: project-reports/2023/2023-Q2-Hyperledger-Cacti.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+1. Major update as merged pull-requests
+    - merging from Cactus and Weaver:
+        - [new_feature] Merge Weaver to Cacti
+            - https://github.com/hyperledger/cacti/commit/4ac560a3222733a2ecf465f80f42b2007ddcb6be
+            - https://github.com/hyperledger/cacti/commit/4171eebd83aba627a155509e13bd95fb50f6ae73
+    - support tools:
+        - [new_feature (WIP)] Cacti GUI app 
+            - https://github.com/hyperledger/cacti/pull/2265
+2. Created a new architecture diagram toward realizing Cacti as a more flexible interoperability toolkit
```
Comment (tkuhrt) [inline project-reports/2023/2023-Q2-Hyperledger-Cacti.md:37]: Can you provide a link to the architecture diagram?

**Review by tkuhrt — CHANGES_REQUESTED** [review]

File: project-reports/2023/2023-Q2-Hyperledger-Cacti.md (hunk 1 above)
Comment (ryjones) [reply project-reports/2023/2023-Q2-Hyperledger-Cacti.md]: fixed

**Review by petermetz — APPROVED** [review]
LGTM (assuming @tkuhrt's change requests are being addressed)

File: project-reports/2023/2023-Q2-Hyperledger-Cacti.md (hunk 2 above)
Comment (VRamakrishna) [reply project-reports/2023/2023-Q2-Hyperledger-Cacti.md:37]: @takeutak Can you add two links: one to the position in `ROADMAP.md` where the architecture diagram is displayed (https://github.com/hyperledger/cacti/blob/main/ROADMAP.md#cacti-v2), and another to the image file (https://github.com/hyperledger/cacti/blob/main/images/cacti-architecture-v1-merge.png)?

**Review by VRamakrishna — APPROVED** [review]

**Review by denyeart — APPROVED** [review]

**Review by lehors — APPROVED** [review]

**Review by bobbi28 — APPROVED** [review]

**Review by swcurran — APPROVED** [review]

**Review by jimthematrix — APPROVED** [review]

File: project-reports/2023/2023-Q2-Hyperledger-Cacti.md (hunk 2 above)
Comment (takeutak) [reply project-reports/2023/2023-Q2-Hyperledger-Cacti.md:37]: @tkuhrt @VRamakrishna added https://github.com/hyperledger/toc/pull/92/commits/1f331d8fd11b9a74c0a030aad6e82e874a1344f7

**Review by tkuhrt — APPROVED** [review]

**Review by mbrandenburger — APPROVED** [review]

**Pull request merged**